IPA.Injector/Injector.cs

```diff
@@ -126,23 +126,16 @@ public static void SetOpenXRRuntime(string[] arguments)
 
             targetRuntime = targetRuntime.ToLower(System.Globalization.CultureInfo.CurrentCulture);
```

Comment:
Please use ordinal comparison whenever possible instead of `ToLower`, as you did below.